src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -62,6 +63,7 @@ internal TestRunner(Options options, ITestExecutor testExecutor)
                             var testResult = await task.ConfigureAwait(false);
                             if (!testResult.Succeeded)
                             {
+                                failures++;
                                 allPassed = false;
```

Comment:
We can remove allPassed now since I don't think there's a case where it's not just the same as `failures > 0`?